tests/test_openclhpp.cpp

```diff
@@ -1665,6 +1665,33 @@ void testKernelSetArgLocal()
     kernelPool[0].setArg(2, cl::Local(123));
 }
 
+void testKernelSetArgBySetKernelArgSVMPointerWithUniquePtrType()
+{
+#if CL_HPP_TARGET_OPENCL_VERSION >= 200
+    std::unique_ptr<int> buffer(new int(1000));
```

Comment:
For the record, debatably this should be `cl::Pointer` instead, especially since we're using `VECTOR_CLASS` for the vector type case, but this needs to pass an explicit `Deleter` so I don't think it's worth it.